CHANGELOG_PENDING.md

```diff
@@ -32,13 +32,15 @@ Friendly reminder: We have a [bug bounty program](https://hackerone.com/tendermi
 - P2P Protocol
 
 - Go API
-  - [logging] \#6534 Removed the existing custom Tendermint logger backed by go-kit. The logging interface, `Logger`, remains.
+  - [node] \#6540 Reduce surface area of the `node` package by making most of the implementation details private. (@tychoish)
```

Comment:
for a few of these, they will require a few sentences in the upgrading.md. 

I know very few people read that 😞 but still worth it.

Reply:
Yeah, I agree. What did you have in mind. The upgrading instructions seem to be compiled for all versions, don't seem to call out every Go API change, which means i think we make the 0.35 changes to the upgrading document in a more holistic manner. and we should probably think about this holistically (and also I don't think we're going to do a better job than the compiler).

Reply:
Breakdown of what is now internal and the new APIs compared to the previous ones. 

How did the node package, what do people need to know in order to upgrade. 

> (and also I don't think we're going to do a better job than the compiler).

huh? We do write api breaking changes in the changelog and upgrading doc even if the compiler helps. Not sure if you are suggesting passing it off to the compiler or not?

Reply:
Yeah I think if in the upgrading doc, we just write a short section on what is now private and the 2-3 new node constructor/APIs, that should suffice.